apps/src/bounce/bounce.js

```diff
@@ -1145,6 +1144,9 @@ Bounce.displayBall = function (i, x, y, rotation) {
                         x * Bounce.SQUARE_SIZE);
   ballIcon.setAttribute('y',
                         y * Bounce.SQUARE_SIZE + Bounce.BALL_Y_OFFSET);
+  var xCenter = (x * Bounce.SQUARE_SIZE) + (Bounce.PEGMAN_WIDTH / 2);
+  var yCenter = (y * Bounce.SQUARE_SIZE) + Bounce.BALL_Y_OFFSET + (Bounce.PEGMAN_HEIGHT / 2);
+  ballIcon.style.transformOrigin = `${xCenter}px ${yCenter}px`;
```

Comment:
Are we sure all browsers implement transformOrigin with pixel coordinates identically?